crates/use_v1/src/main.nr

```diff
@@ -3,7 +3,7 @@ use noir_bigcurve::BigCurve;
 use noir_bigcurve::curves::secp256k1::{Secp256k1, Secp256k1Fr, Secp256k1Scalar};
 use noir_bigcurve::scalar_field::ScalarField;
 
-use plume::plume_v1;
+use plume::{plume_v1, to_field};
```

Comment:
I guess here import shall be `use plume::utils::to_field` to resolve `is private and not visible from the current module` warning.